jest.config.js

```diff
@@ -9,14 +9,19 @@ module.exports = {
   clearMocks: true,
 
   // The directory where Jest should output its coverage files
-  coverageDirectory: "coverage",
+  coverageDirectory: 'coverage',
 
   // The test environment that will be used for testing
-  testEnvironment: "node",
+  testEnvironment: 'jsdom',
 
   // The glob patterns Jest uses to detect test files
   testMatch: [
-    path.resolve("./test/**/?(*.)+(spec|test).[tj]s?(x)")
+    path.resolve('./test/**/?(*.)+(spec|test).[tj]s?(x)')
   ],
 
+  transform: {
+    '^.+\\.js$': 'babel-jest',
+    '^.+\\.svg$': '<rootDir>/test/svg-transformer.js'
```

Comment:
Why has this been added?

Reply:
Okay I think I understand a little about what is going on from the commit message.

But do we really have to carry the code for this ourselves? Can't we use some babel plugin for this? Isn't that what was going on before?

I also can't see here a `.svg` file is being imported.

Reply:
Yes it's not an obvious one this... because this file checks a full page, not just a component, we have the header et al included as part of the output, which means the gov logo. I had a look at a couple of solutions for this, and this one allows us some control over what ends up in the sanpshot - at the moment it's just the filename, but it could also be the full SVG if we thought that were useful.